src/constants/index.js

```diff
@@ -1,5 +1,5 @@
 import { identifiers } from './identifiers';
-import {regexes} from './regexes';
+import { regexes } from './regexes';
```

Comment:
Don't now but for me I think since regexes contains only constants this can be capitalised too?

Reply:
I wouldn't capitalize regexes, since it isn't really a component. (it's just a helper class)